app/views/shared/_email_subscribe_unsubscribe_flash.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if @account_flash.include?("email-subscription-success") %>
+
+  <%= render "govuk_publishing_components/components/success_alert", {
+    message: sanitize("You&rsquo;ve subscribed to emails about this page.")  ,
+    description: sanitize("<p class=\"govuk-body\">Go to your GOV.UK account to <a class=\"govuk-link\" href=\"/email/manage\">see and manage all your GOV.UK email subscriptions</a></p>")
+  } %>
+
+<% elsif @account_flash.include?("email-unsubscribe-success") %>
+
+  <%= render "govuk_publishing_components/components/success_alert", {
+    message: sanitize("You&rsquo;ve unsubscribed from emails about this page."),
+    description: sanitize("<p class=\"govuk-body\">Go to your GOV.UK account to <a class=\"govuk-link\" href=\"/email/manage\">see and manage all your GOV.UK email subscriptions</a></p>")
```

Comment:
For the link, could we please add the `govuk-notification-banner__link` class to make it green?
See this for reference: https://components.publishing.service.gov.uk/component-guide/success_alert/with_styled_link